### PR TITLE
Dr doc refinements

### PR DESCRIPTION
**Pull request by NickGraham101**
### Context
Demoing the DR process to the publish team identified some refinements and fixes that could be made to the DR process.

https://trello.com/c/wx5ad9MF

### Changes proposed in this pull request
- Remove dependency on output from one script to execute another, this maintains compatibility across Linux distributions
- Replaces `grep` with `awk` to maintain compatibility across Linux distributions
- Fixed incorrect file permissions introduced by WSL

### Guidance to review
- The solution was developed on a Windows machine running Ubuntu 20 in WSL.  It should tested on MacOS ideally or another distribution.  A minimal test would be:
  - execute the `deploy-plan` command documented in [recreate-the-lost-postgres-database-instance](https://github.com/DFE-Digital/teacher-training-api/blob/dr-doc-refinements/docs/disaster-recovery.md#recreate-the-lost-postgres-database-instance)
  - execute the commands in [restore-data-from-nightly-backup](https://github.com/DFE-Digital/teacher-training-api/blob/dr-doc-refinements/docs/disaster-recovery.md#restore-data-from-nightly-backup).  This can be done against QA with agreement from the publish team or against the review app by replacing `${paas_env}` on [line 116](https://github.com/DFE-Digital/teacher-training-api/blob/6678c7123f3d0956c067d60cb97cd5328e293561/Makefile#L116) of the make file with `pr-2651`

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: docs/disaster-recovery.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Alert all developers that no one should merge to main branch.
 
 ### Local Dependencies
 
-You will need the [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) and [cf](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html) CLIs installed as well as [make](https://www.gnu.org/software/make/) and either [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) or [tfenv](https://github.com/tfutils/tfenv#installation).
+You will need the [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) and [cf](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html) CLIs installed as well as [jq](https://stedolan.github.io/jq/download/), [make](https://www.gnu.org/software/make/) and either [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) or [tfenv](https://github.com/tfutils/tfenv#installation).
```
Comment (defong) [inline docs/disaster-recovery.md:16]: Can we add it here as well please

https://github.com/DFE-Digital/teacher-training-api/blob/master/.tool-versions

File: docs/disaster-recovery.md [hunk 2]
```diff
@@ -50,26 +50,35 @@ cf delete-service <instance-name>
...
 ```
-make production restore-data-from-nightly-backup CONFIRM_PRODUCTION=YES CONFIRM_RESTORE=YES BACKUP_DATE="yyyy-mm-dd"
+# env is the target environment in the make file e.g. 'production'
+# space is the name of the environment in GOV.UK PaaS, eg 'bat-prod'
+az login
+cf login -o dfe -s <space> -u my.name@digital.education.gov.uk
+make <env> restore-data-from-nightly-backup BACKUP_DATE="yyyy-mm-dd" CONFIRM_RESTORE=YES CONFIRM_PRODUCTION=YES
 ```
 
+##TO DO: change permissions on files chmod +x
```
Comment (defong) [inline docs/disaster-recovery.md:79]: It be good practice to list notable permissions required.

FYI
https://stackoverflow.com/questions/23667859/why-is-git-clone-changing-file-permissions

File: docs/disaster-recovery.md (hunk 1 above)
Comment (NickGraham101) [reply docs/disaster-recovery.md:16]: Added to file

File: docs/disaster-recovery.md (hunk 2 above)
Comment (NickGraham101) [reply docs/disaster-recovery.md]: I've replaced that TO DO with something more useful :)

It looks like the permissions issue might be WSL related, we're still investigating.  File permissions are fixed now though

**Review by merrilmathew — APPROVED** [review]

**Comment by NickGraham101** [comment]
[TT API: simulate data corruption](https://trello.com/c/wx5ad9MF/447-tt-api-simulate-data-corruption)

**Review by saliceti — APPROVED** [review]

**Review by defong — APPROVED** [review]

**Pull request merged**